BrenKt-TRANS-SK-1331-WEB-EmailPro-configuring_Gmail_Android (#6288)
ovh/docs: pages/web_cloud/email_and_collaborative_solutions/email_pro/how_to_configure_android/guide.pt-pt.md

--- 
title: "E-mail Pro - Configurar um endereço de e-mail no Gmail para Android"
excerpt: "Aprenda a configurar uma conta E-mail Pro no Android, através da aplicação Gmail"
updated: 2024-03-15
--- 

<style>
.w-400 {
max-width:400px!important;
}
.h-600 {
max-height:600px!important;
}
</style>

## Objetivo

Os endereços de e-mail do serviço E-mail Pro podem ser configurados em vários softwares de e-mail compatíveis. Isto permite-lhe enviar e receber e-mails no dispositivo que preferir. Encontre neste guia as etapas de configuração de um endereço de e-mail E-mail Pro a partir da aplicação Gmail presente nos dispositivos Android.

**Saiba como configurar uma conta E-mail Pro no Android através da aplicação Gmail.**

> [!warning]
>
> A OVHcloud oferece-lhe serviços cuja configuração, gestão e responsabilidade é da sua responsabilidade. Assim, deverá assegurar o seu bom funcionamento.
>

> > Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção « Ir mais longe » deste guia.

Reviewer: ```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](/links/partner) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, consulte a secção « Ir mais longe » deste guia.
```

Same link can be replaced on other translations

## Requisitos

> - Ter um plano [E-mail Pro](https://www.ovhcloud.com/pt/emails/email-pro/).

Reviewer: ```suggestion
- Ter um plano [E-mail Pro](/links/email-pro).
```

>> ![emailpro](images/emailpro-android-06.png){.thumbnail .h-600}
>>
> **Etapa 7**
>> Escolha a frequência com que pretende que os seus e-mails fiquem sincronizados.<br><br>
>>![emailpro](images/emailpro-android-07.png){.thumbnail .h-600}
>>
> **Etapa 8**
>> Determine o nome a ser apresentado do seu endereço de e-mail na aplicação Gmail e prima a tecla "Seguinte" {.action}.<br><br>
>> ![emailpro](images/emailpro-android-08.png){.thumbnail .h-600}
>>

Depois de configurar o endereço de e-mail, já só precisa de o utilizar ! Pode desde já enviar e receber mensagens a partir da sua aplicação Gmail.

> [!success]
>
> A OVHcloud disponibiliza uma aplicação web que pode utilizar para aceder ao seu e-mail diretamente a partir do browser da Web: <https://www.ovhcloud.com/pt/mail/>. Para aceder, só precisa dos dados de acesso do seu endereço de e-mail.

### Definições POP, IMAP e SMTP

> Para a receção dos e-mails, ao escolher o tipo de conta, recomendamos uma utilização em **IMAP**. No entanto, pode selecionar **POP**. Para compreender o funcionamento respetivo, consulte a nossa secção [« POP ou IMAP, qual é a diferença? »](#popimap)

Reviewer: ```suggestion
Para a receção dos e-mails, ao escolher o tipo de conta, recomendamos uma utilização em **IMAP**. No entanto, pode selecionar **POP**. Para compreender o funcionamento respetivo, consulte a nossa secção [POP ou IMAP, qual é a diferença?](#popimap)
```
